com.slugchess.SlugChess.yml

```diff
@@ -0,0 +1,61 @@
+app-id: com.slugchess.SlugChess
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: 'slugchess-run.sh'
+sdk-extensions: 
+  - org.freedesktop.Sdk.Extension.dotnet7
+build-options: 
+  append-path: '/usr/lib/sdk/dotnet7/bin'
+  append-ld-library-path: '/usr/lib/sdk/dotnet7/lib'
+  prepend-pkg-config-path: '/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/dotnet7/lib/pkgconfig'
+  env: 
+    DOTNET_CLI_TELEMETRY_OPTOUT: 'true'
+  arch:
+#    arm: 
+#      env: 
+#        RUNTIME: linux-arm
+#    aarch64: 
+#      env: 
+#        RUNTIME: linux-arm64
+    x86_64: 
+      env: 
+        RUNTIME: linux-x64
+finish-args:
+  # X11
+  - --socket=fallback-x11
```

Comment:
--share=ipc is needed for x11 (and fallback)

Reply:
I have added `--share=ipc`